src/EditorFeatures/CSharpTest/InvertConditional/InvertConditionalTests_FixAll.cs

```diff
@@ -0,0 +1,412 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.InvertConditional
+{
+    [Trait(Traits.Feature, Traits.Features.CodeActionsInvertConditional)]
+    public partial class InvertConditionalTests : AbstractCSharpCodeActionTest
+    {
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsFixAllOccurrences)]
+        public async Task InvertConditional_FixAllInDocument()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M(bool x, int a, int b)
+    {
+        var c = {|FixAllInDocument:|}x ? a : b;
```

Comment:
how did you determine which features would get fix all?  i'm fairly skeptical about the value in something like this...

Reply:
Just the most easiest one I could find :-). I am going to sync with @mikadumont once this PR goes in so we can prioritize refactorings that should get FixAll support.